src/early_multicellular_stage/editor/EarlyMulticellularEditor.cs

```diff
@@ -106,7 +106,8 @@ public override void SetEditorObjectVisibility(bool shown)
 
     public void OnCurrentPatchUpdated(Patch patch)
     {
-        cellEditorTab.CalculateOrganelleEffectivenessInPatch(patch);
+        cellEditorTab.SetPatchConditions(patch);
+        cellEditorTab.CalculateOrganelleEffectivenessInPatch();
```

Comment:
I think rather than modifying CalculateOrganelleEffectivenessInPatch to no longer take a patch (the hint is in the name that it should absolutely take a patch as a parameter) and adding a new method, one higher level method for applying patch conditions should be made and called here.